src/content/docs/integrations/react-native.mdx

```diff
@@ -28,14 +28,17 @@ In React Native, MSW works similarly to Node.js, but omits certain request inter
 MSW relies on standard JavaScript classes that are not present in React Native, like `URL`. Please install the polyfills below to guarantee proper MSW execution in React Native.
 
 ```sh
-npm install react-native-url-polyfill
+npm install react-native-url-polyfill text-encoding
 ```
 
 Create a new `msw.polyfills.js` file with the following content:
 
 ```js
 // msw.polyfills.js
 import 'react-native-url-polyfill/auto'
+import { TextEncoder } from 'text-encoding';
```

Comment:
Should we do the same for `TextDecoder`?

Reply:
Maybe

I'm actually getting an empty response body with my mocked URLs even after using this polyfill

Would be willing to walk through things over a call if you're able. 

I went down a rabbit hole and found it might have to do with this file from this library
https://github.com/mswjs/msw/blob/657edd8b4e24043614ea0eed97ce4987df6009e7/src/native/index.ts#L1C39-L1C39

I don't think it works as expected in react native's Hermes engine.

Reply:
same error me

Reply:
@danieloi, looks like you linked the wrong file (?). Could you please double-check?